app/webpacker/components/Panel/pages/RegionManager/index.jsx

```diff
@@ -22,7 +22,9 @@ const defaultRegion = {
   friendlyId: '',
 };
 
-function UserGroupVisibility({ userGroup, save, sync }) {
+function UserGroupVisibility({
+  userGroup, save, sync, setSaveError,
```

Comment:
I don't see any code change where components pass `setSaveError`...
Where are you passing any value along?

Reply:
It's passed in line 134 and 169, but I guess you already saw that, did you mean something else?

Reply:
Yeah but where is that `setSaveError` that is being passed in those lines actually defined?
Was it already existing before?

Reply:
Yes, it was already existing earlier. But it was not passing the error to the `Errored` component. I've done it now. Still it's not displaying a proper message, but I think that is fine.